### ISSUE TITLE
Disable scroll wheel value change on panel

### ISSUE THREAD

**Issue by Balls0fSteel** [louislam/uptime-kuma#1796]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

This should be super simple to implement, test, deploy.

When you ADD:
- You can edit Port number with scroll.

When you EDIT:
- You can change both Port, and Heartbeat values with scroll wheel.

### ✔️ Solution

**Disable this behavior.**
While moving around with the mouse, it's extremely simple to change these values.

According to this, you can use "tel" instead of "number" input type:
https://stackoverflow.com/questions/9712295/disable-scrolling-on-input-type-number

I checked it, worked for me (disabled scroll) but of course, more testing is needed I believe.

**Comment by CommanderStorm** [comment]
@Balls0fSteel 
Why is not allowing users to use the scroll wheel on these inputs important/better?

**Comment by OpenSpaghettiSauce** [comment]
@CommanderStorm Because when you scroll through the settings, you can accidentally change the port number or other number resulting in a headache. That's about it.

**Comment by OpenSpaghettiSauce** [comment]
Yeah up to you guys. I just often go through monitors to double-check and it's very easy to mess something up due to this. I remember Qt also had fields like this and qBittorrent would change values as I scrolled settings.

I don't know, you guys know best. Thanks for your hard work btw!

**Comment by luckman212** [comment]
I was about to open a new issue but then I found this. 2+ years later and we are now at v2 but this just "bit" me. It's dangerous because the changes are subtle and I didn't even notice that it had happened until my alarm didn't fire because the "retries" field had switched from `0` to `6` during casual mousing...

Another vote to please fix this or at least make it a setting! @OpenSpaghettiSauce @Balls0fSteel @CommanderStorm
